cmd/config/checks.go

```diff
@@ -68,6 +78,9 @@ func NewCheckSuite(metadataClient voucher.MetadataClient, names ...string) (*vou
 	scanner := newScanner(metadataClient, auth)
 	checksuite := voucher.NewSuite()
 
+	trustedBuilderIdentities := voucher.ToMapStringBool(viper.GetStringMap("trusted-builder-identities"))
```

Comment:
As per one of my other change requests this should change to `GetStringSlice` I believe!
```suggestion
	trustedBuilderIdentities := viper.GetStringSlice("trusted-builder-identities")
```

Reply:
Done.